### PR TITLE
feat(users-section):implement users section

### PR DESCRIPTION
**Pull request by LeVHoangduc**

### CODE REVIEW

File: practice-one/src/assets/css/users.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+}
+.users__list {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 20px;
+  padding-top: 65px;
+}
+.user {
+  box-sizing: border-box;
+  flex-basis: 358.138px;
```
Comment (hungnguyenhuu) [inline practice-one/src/assets/css/users.css:12]: Could you help to check this number? It looks so weird.

**Comment by hungnguyenhuu** [comment]
@LeVHoangduc Just a little comment. Pls help check

File: practice-one/src/assets/css/users.css (hunk 1 above)
Comment (LeVHoangduc) [reply practice-one/src/assets/css/users.css:12]: i will check it

Comment (LeVHoangduc) [reply practice-one/src/assets/css/users.css]: @hungnguyenhuu  i have updated it again. Please help to check it

**Pull request merged**